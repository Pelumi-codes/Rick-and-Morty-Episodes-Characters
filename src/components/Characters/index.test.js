import React from "react";
import renderer from "react-test-renderer";

import Character from "./index";
it("renders list of characters", () => {
  const tree = renderer.create(<Character />).toJSON();
  expect(tree).toMatchSnapshot();
});