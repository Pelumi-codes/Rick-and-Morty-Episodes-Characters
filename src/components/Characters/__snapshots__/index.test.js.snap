// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders list of characters 1`] = `
<div
  className="css-0"
>
  <div
    className="css-1dnz8oy"
  >
    <div
      className="css-2imjyh"
    />
  </div>
</div>
`;
